forc-plugins/forc-doc/src/render.rs

```diff
@@ -161,19 +161,41 @@ fn page_from(rendered_content: Box<dyn RenderBox>) -> String {
 }
 
 /// Basic HTML header component
-fn html_head(location: String, decl_ty: String, decl_name: String) -> Box<dyn RenderBox> {
+fn html_head(
+    module_depth: usize,
+    location: String,
+    decl_ty: String,
+    decl_name: String,
+) -> Box<dyn RenderBox> {
+    // TODO: Find a way to dynamically create paths based on module_depth.
+    // This also applies to html_body(), sidebar() and navigation between pages as a whole.
```

Comment:
I am not much in the loop with these parts of the code so sorry if this is something obvious 😅 Do you need to add preceding `../` based on module depth (for example for `module depth = 2` -> `../../normalize.css` while `module_depth = 3` -> `../../../normalize.css`) if that is the case maybe we can collect the preceding `../` first and add "normalize.css" part after that. But it feels like this might be a little bit more involved 🤷‍♂️

Reply:
I think that gives me an idea, thanks Kaya!